### ISSUE TITLE
Users are not able to access Sort and Filter functionality present for column headers via keyboard:A11y_MSSQL for VSCode_Connection database_Adding New Query_Keyboard.

### ISSUE THREAD

**Issue by ganagdhar123** [microsoft/vscode-mssql#19821]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### GitHub Tags:
#A11yTCS;#MSSQL for VSCode;#SH_MSSQLForVSCode_Win32_Jul2025;#DesktopApp;#FTP;#Win11;#A11ySev2;#WCAG1.4.3;#WCAG2.1.1;#A11yMAS;
### Environment Details:
Application: MSSQL for VSCode
Visual studio code version: 1.102.1
OS: Windows 11 Enterprise 24H2
Build: 26100.4652
### Repro Steps: 

1. Launch visual studio code application
2. Sign in with "GitHub" account
3. Tab till "Extension" and press enter
4. Tab till search box and enter "SQL" (mssql) server and install extension
5. Tab till any database and click on "New query" and press enter.
6. write the new query and run the query.
7. The output will be displayed
8. Observe that the users are not able to access Sort and Filter functionality present for column headers via keyboard

### Actual:
Users are not able to access Sort and Filter functionality present for column headers via keyboard
### Expected:
Users should be able to access and operate Sort and Filter controls on column headers using keyboard navigation alone
### User Impact:
Keyboard-only users and those using assistive technologies like screen readers are unable to sort or filter data in tables.
### Attachment:

https://github.com/user-attachments/assets/98509efc-f238-4921-b40c-acfa3d449d42

**Comment by aasimkhan30** [comment]
F3 will be mapped to sort and filter menu
